site/src/pages/__tests__/search.js

```diff
@@ -128,4 +135,21 @@ describe("Search", () => {
       "this is my <strong>beautiful</strong> <strong>description</strong>"
     )
   })
+
+  it("the query sanitizer is called with the query when updateSearchResults is called", () => {
+    const evt = { target: { value: "abc" } }
+    var index = new Index()
+    ;["title", "author", "tags", "description", "body"].forEach(name =>
+      index.addField(name)
+    )
+    const data = {
+      allMarkdownRemark: articleList,
+      siteSearchIndex: { index: index.toJSON() }
+    }
+    // const tree = renderer.create(<Search data={data} />).toJSON()
+
+    const search = renderer.create(<Search data={data} debounceDelay={0} />)
+    search.getInstance().updateSearchResults(evt)
+    expect(QuerySanitizer).toHaveBeenCalledTimes(1)
```

Comment:
Better to test it has been called 0 times before updateSearchResults(), then verify the method is called exactly once with the string "abc" afterwards.  As this test stands there is no way to validate which call (render() or update...()) has caused the method invocation, or with what data.

Reply:
done